pkg/controller/controller_binding_test.go

```diff
@@ -241,8 +242,12 @@ func TestReconcileServiceBindingNonExistingClusterServiceClass(t *testing.T) {
 	events := getRecordedEvents(testController)
 	assertNumEvents(t, events, 1)
 
-	expectedEvent := corev1.EventTypeWarning + " " + errorNonexistentClusterServiceClassMessage + " " + "References a non-existent ClusterServiceClass (K8S: \"nosuchclassid\" ExternalName: \"" + testNonExistentClusterServiceClassName + "\")"
-	if err := checkEvents(events, []string{expectedEvent}); err != nil {
+	expectedEvent := new(MessageBuilder).warning().reason(errorNonexistentClusterServiceClassMessage)
```

Comment:
I am liking where this is heading - one thought -

How about instead of:

```go
expectedEvent := new(MessageBuilder).warning().reason(errorNonexistentClusterServiceClassMessage)
```

we have something like:

```go
expectedEvent := warningEvent().reason(errorNonexistentClusterServiceClassMessage)
```

Reply:
Like a function in the controller test that wraps and returns the `new(MessageBuilder).warning()`

how about `warningEventBuilder().yada...` to make it clear it is still this builder object?

Reply:
I could live with `warningEventBuilder()` over `new(MessageBuilder).warning()`